utils/sane-backends/Makefile

```diff
@@ -134,6 +137,11 @@ $(call Package/sane-backends/Default/description)
 This package contains the SANE daemon.
 endef
 
+define Package/sane-daemon/conffiles
```

Comment:
Why this define was moved? It was after 'define Package/sane-daemon/install'
If you have a reason, we can keep it here but it is better to avoid moving code.